src/data/nat/basic.lean

```diff
@@ -452,6 +452,13 @@ protected lemma div_eq_zero_iff {a b : ℕ} (hb : 0 < b) : a / b = 0 ↔ a < b :
   λ h, by rw [← nat.mul_left_inj hb, ← @add_left_cancel_iff _ _ (a % b), mod_add_div,
     mod_eq_of_lt h, mul_zero, add_zero]⟩
 
+lemma eq_zero_of_div_eq_zero {a b : ℕ} (h : a / b = 0) (w : b ∣ a) (hb : 0 < b) : a = 0 :=
```

Comment:
This should go next to `eq_of_dvd_quot_one`, have a consistent name and take its arguments in the same order. `hb` is redundant; the proof can be `by rw [←nat.div_mul_cancel w, h, zero_mul]`. A similar proof should work for `eq_of_dvd_quot_one`.

Reply:
Thanks! I've made them uniform.

Reply:
Shall I separate these out to a second PR, now that Chris' proof doesn't use them?